### ISSUE TITLE
No module named 'debugging'

### ISSUE THREAD

**Issue by Guodonggogo** [Jerrypiglet/ScaleNet#1]
it seems that debugging module is not uploaded?

**Comment by Jerrypiglet** [comment]
Hey @Guodonggogo yes it looks like the module is missing. Here it is https://gist.github.com/Jerrypiglet/643ce55c65aae34a911e56a8748ec9c0. I will update the script to the the repo within next week and let you know. Also please let me know if you encounter other issues!

**Comment by Guodonggogo** [comment]
Thx a lot. It seems under "/panorama_cropping_dataset_generation/debugging.py", I manually set the path, it works now. Thx, really awesome work. But don't know how to output object height as your paper shown. Waiting for your scripts anyway.

**Comment by Jerrypiglet** [comment]
@Guodonggogo Yep. The code for the full scale estimation will be uploaded soon (hopefully within two weeks). Thanks for your patience!

**Comment by Jerrypiglet** [comment]
> it seems that debugging module is not uploaded?

Hey Guodong, the repo has just been updated with demo of inference and virtual object insertion using the ScaleNet on COCO images. I will add training codes to the repo by this week, but if you only need inference you can have a try right now. Sorry again for late upload and please let me know of any issues...

**Comment by grainw** [comment]
@Jerrypiglet Thanks for your contribution. I'm very interested in your research. Can you tell me when you can share the training code.

**Comment by Jerrypiglet** [comment]
@grainw Hey Alex, sorry for late release of the training part. If you need the code for ICCV or other pending deadlines I can try to find some time to make it done soon; otherwise I will work on it after the ICCV deadline on Mar 24. Let me know if you need it very soon.

**Comment by grainw** [comment]
@Jerrypiglet Thanks for your response. if you have free time before the ICCV deadline, I hope you can release the training code as soon as possible. I am so interested in this research that I want to conduct some experiments as soon as possible.

**Comment by wmrenr** [comment]
@Jerrypiglet Thanks for your contribution. I'm very interested in your research. When will the training code for scale estimation be released?

**Comment by cindy-J** [comment]
@Jerrypiglet I also encounter the same problem, could you please point out the debugging module ？
